packages/style-utilities/README.md

```diff
@@ -1,25 +1,25 @@
-# @uifabric/styling
+# @fluentui/style-utilities
 
 **Styling helpers for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 ([formerly Office UI Fabric React](https://developer.microsoft.com/en-us/office/blogs/ui-fabric-is-evolving-into-fluent-ui/))
 
-## Using the styling package
+## Using this package
 
 Integrating components into your project depends heavily on your setup. The recommended setup is to use a bundler such as webpack which can resolve NPM package imports in your code and can bundle the specific things you import.
 
-If you're using `@fluentui/react`, the `@uifabric/styling` package contents are re-exported under `@fluentui/react/lib/Styling`. It's recommended to access styling this way rather than via a direct dependency.
+If you're using `@fluentui/react`, the `@fluentui/style-utilities` package contents are re-exported under `@fluentui/react/lib/Styling`. It's recommended to access the package this way rather than via a direct dependency.
 
-In a project which doesn't use `@fluentui/react`, you can still install the styling package as a dependency:
+In a project which doesn't use `@fluentui/react`, you can still install this package as a dependency:
 
-```bash
-npm install --save @uifabric/styling
+```
+npm install --save @fluentui/style-utilities
 ```
 
 This gives you access to styling-related constants, utilities, and Fabric Core style classes through JavaScript.
 
 ## Overriding the theme colors
```

Comment:
could you help remove this section? since we are trying to move theme related out of this package. the reason why `loadTheme` is still here is ... we don't have a better place for it. 
User can find information about it in our [wiki](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components#loadtheme). So we don't need to call-out here